### ISSUE TITLE
Transfer Learning: apply a smaller learning rate to the pre-trained part of the model

### ISSUE THREAD

**Issue by rojberr** [GHOST-Science-Club/tree-classification-irim#16]
Finetune the convnet with a smaller learning rate for the pre-trained convolutional part of the model.

[Idea](https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html#finetuning-the-convnet).
